environment/config.go

```diff
@@ -204,9 +207,11 @@ func newStaticShardSet(numShards int, listenAddress, hostID string) (sharding.Sh
 		return nil, nil, err
```

Comment:
Actually btw, the hash fn needs to be `sharding.DefaultHashFn(len(shards))` because it's creating a hashing function that needs to know how many shards its sharding over.